### ISSUE TITLE
Modules listed in package.json don't exist in bundled package

### ISSUE THREAD

**Issue by ryanwilsonperkin** [lbenie/ts-jest-mock#984]
Looking at the most recent version of `ts-jest-mock`, the `dist/` directory contains the following:

```
types/
ts-jest-mock.js
ts-jest-mock.umd.cjs
```

This does not match with the values listed in `package.json`:
```json
{
  "main": "./dist/ts-jest-mock.umd.js",
  "module": "./dist/ts-jest-mock.es.js",
  "exports": {
    ".": {
      "types": "./dist/types/index.d.ts",
      "import": "./dist/ts-jest-mock.es.js",
      "require": "./dist/ts-jest-mock.umd.js"
    }
  },
}
```

and leads to a module resolution error when importing `ts-jest-mock`

**Comment by lbenie** [comment]
@ryanwilsonperkin I'm releasing a new version, forgot to rename these when I released as an es module

**Comment by lbenie** [comment]
fixed in 1.4.1
